roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -201,7 +213,11 @@ class Namespace(object):
 
 
 class ResetNeutron(object):
-    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-linuxbridge-agent']
+    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-agent', 'neutron-linuxbridge-agent']
```

Comment:
I do not think these agents get renamed. I do not see neutron-metadata-proxy as a neutron service in Kilo. I thought this was just a typo?